silverstripe/framework/SS-2015-026-1.yaml

```diff
@@ -4,7 +4,7 @@ cve:       ~
 branches:
     3.0.x:
         time:     2015-11-11 14:31:00
-        versions: [>=3.0.0,<=3.0.14]
+        versions: [>=3.0.0,<=3.0.15]
```

Comment:
What about simply using `<3.1.0` here? It seems to me that `3.0.x` versions don't receive any fixes at all.